README.md

```diff
@@ -96,4 +105,4 @@ ddcutil --brief detect > $XDG_CACHE_HOME/ddcutil_detect
 This extension is developed and maintained by [@datij](https://github.com/datij)
 
 #### Thanks to the following people for contributing via pull requests:
-- @oscfdezdz for adding new settings ui, keyboard shortcuts and ability to set icon location
\ No newline at end of file
+- @oscfdezdz for adding new settings ui, keyboard shortcuts and ability to set icon location
```

Comment:
Nothing here changed. I used GitHub's built-in editor for this PR, so it may have done some automatic whitespace changes.